cmd/root.go

```diff
@@ -106,5 +106,5 @@ func initClient() {
 	}
 
 	client = govultr.NewClient(nil, key)
-	client.RateLimit = 1 * time.Millisecond
+	client.SetRateLimit(1 * time.Millisecond)
```

Comment:
Are you sure this is correct? This would basically eliminate the retries, as it would try 3 times (the default retry max), with delays of under 1ms between each request. Unless I'm mistaken, you probably mean 1 second, not millisecond.

```suggestion
	client.SetRateLimit(1 * time.Second)
```